pkg/inventory/inventory.go

```diff
@@ -110,13 +109,13 @@ func SplitUnstructureds(objs []*unstructured.Unstructured) (*unstructured.Unstru
 		}
 	}
 	if len(invs) == 0 {
-		return nil, resources, NoInventoryObjError{}
+		return nil, resources, nil
 	} else if len(invs) > 1 {
 		return nil, resources, MultipleInventoryObjError{
 			InventoryObjectTemplates: invs,
 		}
 	}
-	return invs[0], resources, nil
+	return WrapInventoryInfoObj(invs[0]), resources, nil
```

Comment:
This change makes this function not usable for those who want to use a custom inventory object. Is this intentional?